msal/src/androidTest/java/com/microsoft/identity/client/PublicClientApplicationTest.java

```diff
@@ -127,189 +103,177 @@ public void tearDown() {
         Telemetry.disableForTest(false);
     }
 
+
+    @Test
+    public void testSingleAccountConstructor() {
+        final Context context = new PublicClientApplicationTest.MockContext(mAppContext);
+        mockPackageManagerWithDefaultFlag(context);
+        mockHasCustomTabRedirect(context);
+
+        try {
+            ISingleAccountPublicClientApplication app = PublicClientApplication.createSingleAccountPublicClientApplication(context, com.microsoft.identity.msal.test.R.raw.test_msal_config_single_account);
+            Assert.assertTrue(app instanceof ISingleAccountPublicClientApplication);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        } catch (MsalException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Test
+    public void testMultipleAccountConstructor() {
+        final Context context = new PublicClientApplicationTest.MockContext(mAppContext);
+        mockPackageManagerWithDefaultFlag(context);
+        mockHasCustomTabRedirect(context);
+
+        try {
+            IMultipleAccountPublicClientApplication app = PublicClientApplication.createMultipleAccountPublicClientApplication(context, com.microsoft.identity.msal.test.R.raw.test_msal_config_multiple_account);
+            Assert.assertTrue(app instanceof IMultipleAccountPublicClientApplication);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        } catch (MsalException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Test
+    public void testMultipleAccountAsyncConstructor() {
+        final Context context = new PublicClientApplicationTest.MockContext(mAppContext);
+        mockPackageManagerWithDefaultFlag(context);
+        mockHasCustomTabRedirect(context);
+
+        PublicClientApplication.createMultipleAccountPublicClientApplication(context, com.microsoft.identity.msal.test.R.raw.test_msal_config_multiple_account, new PublicClientApplication.IMultipleAccountApplicationCreatedListener() {
```

Comment:
I would suggest importing  `com.microsoft.identity.msal.test.R` at the top, and then in the method calls we can just do R.raw.filename. So this way our statements aren't too long and don't need to scroll horizontally. 

Same applies for all such occurrences in this file.